pravega-k8-log-collection/pravega-k8s-log.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+mkdir -p /tmp/logdir
+output_dir="/tmp/logdir"
+kubectl get po,svc,pvc --all-namespaces > $output_dir/kubectl-get-po-svc-pvc-all-namespace_output
```

Comment:
Use `kubectl get all -l app=pravega-cluster --all-namespaces` to list all `PravegaCluster` resources.

Reply:
Current command is taking pod, service and pvc details from the cluster and I thought of using these in Nautilus log collection as well, then only change is pravege filter which I am currently using. Let me check the o/p of your suggestion if all components it is taking will be updating the same, else would like keep the current one.

Reply:
Seems to me `kubectl get all -l app=pravega-cluster --all-namespaces` is providing limited o/p from cluster (only bookkeeper, segmentstore and controller pod, bookie and controller services / replicaset / statefulset info are visible. The view from `kubectl get po,svc,pvc --all-namespaces`  is more elaborative and taking care of zookeeper info along with more services info and persistentvolumeclaims. I would like to keep this view and if Nautilus platform would like to use this utility changes required to accommodate those areas will be minimal.